src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -263,15 +258,20 @@ private boolean isBitrateImproving(int newBitrateBps)
      * @return the restored packet.
      */
     @Override
-    public RawPacket reverseTransform(
-            RawPacket packet)
+    public void incomingPacket(RawPacket packet)
     {
-        logger.info("Using RemoteBitrateEstimatorAbsSendTime: Instantiating.");
+        int extensionID = this.extensionID;
+        if (extensionID != -1)
+        {
+            long ast =
+                AbsSendTimeEngine.getAbsSendTime(packet, (byte) extensionID);
 
-        incomingPacketInfo(System.currentTimeMillis(), absoluteSendTimeEngine
-                .getAbsSendTime(packet), packet.getPayloadLength(),
-                packet.getSSRCAsLong());
-        return packet;
+            if (ast != -1)
+            {
+                incomingPacketInfo(System.currentTimeMillis(), ast,
```

Comment:
the name of this method seems odd to me, can we change it to a verb?  `incomingPacketInfo` sounds like a type instead of a method (`processIncomingPacketInfo`?)